### ISSUE TITLE
If a Suggested Location's website doesn't include `http://`, then prepend it to it

### ISSUE THREAD

**Issue by RyanTG** [scottwainstock/pbm#803]
A lot of Suggested Locations (SL) are submitted with website links that look like `www.website.com` or `website.com`. 

I'm guessing the submitters are either copy/pasting the _text_ from google maps (rather than the hyperlink), or maybe when they're on mobile, copy/pasting the url in a browser doesn't grab the http://. 

This is an issue because all of our website links must begin with `http://`.

Many SLs look great aside from this one field. It then requires the admin to edit the SL just to fix this field before they can Promote it. It would be cool if we could automatically detect when websites are missing the `http://` and then prepend `http://` to it.

So basically, IF website field has content, but doesn't start with http://, THEN add http:// to beginning of string.

**Comment by RyanTG** [comment]
I think this will be too messy. Sometimes people submit websites that look like: "i love lucy.com" So prepending http:// wouldn't help anything there.
